requirements.txt

```diff
@@ -0,0 +1,9 @@
+caniusepython3==4.0.0
+edx-lint==0.5.1
+isort==4.2.5
+pycodestyle==2.0.0
+pydocstyle==1.1.1
+pytest==3.0.3
+pytz
+six==1.10.0
+tox==2.3.1
```

Comment:
The only actual app dependency at the moment is pytz, so we'll hold off on pip-tools

tox-battery will be useful.  Adding.